### ISSUE TITLE
Which LED Name Badge to buy are compatible?

### ISSUE THREAD

**Issue by Sylvain303** [olell/badgemagic-firmware#3]
Hello @olell 

Did you find some compatible badge to buy?

I'm animating a workshop in France for: making our own badge.

I bought some magic badge at last FOSDEM.

I would like to know where I can buy more compatible badges?
Compatible with the Android app and the Bluetooth.
Do you have some hint?

We are a French LUG at Annecy, https://agu3l.org/

Regards,
Sylvain

**Comment by olell** [comment]
Hello Sylvain,
thanks for you request. I bought my badge also from the people over at fossasia, who meanwhile also maintain a custom firmware for those badges, take a look at their repository https://github.com/fossasia/badgemagic-firmware

Around a year ago I researched some links to buy more of those badges, and I found a lot on aliexpress by searching for "bluetooth led nametag", the firmware in this repository and I guess also in the other targets the 11x44 pixel versions with bluetooth LE. Here are some example links: (I am not affiliated with any of those)
https://de.aliexpress.com/item/1005003823059543.html
https://de.aliexpress.com/item/1005004406683122.html
https://de.aliexpress.com/item/1005004921988054.html

I hope that answers your question :) 
olel

**Comment by Sylvain303** [comment]
Thanks @olell,

Did you try to flash some badge yourself?

I've read I must desoldering the battery before flashing a badge. (https://github.com/fossasia/badgemagic-firmware/issues/37)

I also found some similar reference on aliexpress too. I was just unsure I could make it work with a FOSSASIA BLE app.

I could buy all 3 references, and do some test. 🤔

Regards,
Sylvain.

**Comment by olell** [comment]
Yes, I successfully flashed my badge using the firmware from this repository. I haven't tried the fossasia firmware, but I don't think that should be a problem. 
I'm pretty sure that as long as the badges look the same, they are the same. The only thing to note is that there are two different resolutions available, and this repo targets the 11x44. 
In my README.md file you can also find a picture of the PCB, including a marking code. If your PCB has the same marking, you should be fine. The README.md file also contains some instructions on how to check that the WCHISP tool detects the board correctly and how to flash it. But yes, it's true that you have to desolder the battery, otherwise it's not possible to completely power down the controller.
